src/compiler/moduleNameResolver.ts

```diff
@@ -1,4 +1,8 @@
 namespace ts {
+
+    const _fs: typeof import("fs") = require("fs");

```

Comment:
A real implementation would plumb this through the host and handle the case where it's missing.